### ISSUE TITLE
version 2.0.3-r11 /opt/bitnami/discourse-sidekiq/config/discourse.conf  hostname = undefined

### ISSUE THREAD

**Issue by rene-gomez** [bitnami/bitnami-docker-discourse#48]
Description

when signing a new user, the server send an email to confirm and activate the account, the server send an email with URL undefined

```

Welcome to My site!!

Click the following link to confirm and activate your new account:
http://undefined/u/activate-account/45f78142d1e6a06bf82673abd08470e4
```

Steps to reproduce the issue:

```
[First Step]
create a new account
[Second Step]
wait a email confirmation
[and so on...]
the confirmation arrives, with URL undefined
```

In my YAML file I declare the ENV in two containers, the bitnami and sidekiq:

```
version: '2'
services:
  postgresql:
    image: 'bitnami/postgresql:9.6'
    volumes:
      - 'postgresql_data:/bitnami'
  redis:
    image: 'bitnami/redis'
    environment:
      - ALLOW_EMPTY_PASSWORD=yes
    volumes:
      - 'redis_data:/bitnami'
  discourse:
    image: 'bitnami/discourse'
    labels:
      kompose.service.type: nodeport
    ports:
      - '3000'
    depends_on:
      - postgresql
      - redis
    volumes:
      - 'discourse_data:/bitnami'
    environment:
      - DISCOURSE_EMAIL=email@gmail.com
      - DISCOURSE_HOSTNAME=forum.anyname.com
      - DISCOURSE_SITENAME=forum.anyname.com
      - POSTGRESQL_HOST=postgresql
      - POSTGRESQL_ROOT_USER=postgres
      - POSTGRESQL_CLIENT_CREATE_DATABASE_NAME=bitnami_application
      - POSTGRESQL_CLIENT_CREATE_DATABASE_USERNAME=bn_discourse
      - POSTGRESQL_CLIENT_CREATE_DATABASE_PASSWORD=bitnami1
      - DISCOURSE_POSTGRESQL_NAME=bitnami_application
      - DISCOURSE_POSTGRESQL_USERNAME=bn_discourse
      - DISCOURSE_POSTGRESQL_PASSWORD=bitnami1
      - SMTP_HOST=smtp.gmail.com
      - SMTP_PORT=587
      - SMTP_DOMAIN=forum.anyname.com
      - SMTP_USER=email@gmail.com
      - SMTP_PASSWORD=password
  sidekiq:
    image: 'bitnami/discourse'
    depends_on:
      - discourse
    volumes:
      - 'sidekiq_data:/bitnami'
    command: 'nami start --foreground discourse-sidekiq'
    environment:
      - DISCOURSE_POSTGRESQL_NAME=bitnami_application
      - DISCOURSE_POSTGRESQL_USERNAME=bn_discourse
      - DISCOURSE_POSTGRESQL_PASSWORD=bitnami1
      - DISCOURSE_HOST=discourse
      - DISCOURSE_PORT=3000
      - SMTP_HOST=smtp.gmail.com
      - SMTP_PORT=587
      - SMTP_USERemail@gmail.com
      - SMTP_PASSWORD=password
      - DISCOURSE_HOSTNAME=forum.anyname.com
      - HOSTNAME=forum.anyname.com
volumes:
  postgresql_data:
    driver: local
  redis_data:
    driver: local
  discourse_data:
    driver: local
  sidekiq_data:
    driver: local
```

Afert UP and run, I can see the discourse.conf at sidekiq container

```
root@6e2d67c555a5:/# cat  /opt/bitnami/discourse-sidekiq/conf/discourse.conf

  db_name = bitnami_application
  db_host = postgresql
  db_port = 5432
  db_password = bitnami1
  db_username = bn_discourse
  db_pool = 25

  hostname = undefined
  redis_port = 6379
  redis_host = redis
  redis_password =
  smtp_address = smtp.gmail.com
    smtp_port = 587
    smtp_domain = example.com
    smtp_user_name = email@gmail.com
    smtp_password =  password
    smtp_enable_start_tls = true
    smtp_authentication = login

```
So the ENV vars does not work

**Comment by beltran-rubo** [comment]
Thanks for reporting it, we are looking into it.

**Comment by beltran-rubo** [comment]
hi @rene-gomez this issue has been fixed in the latest 2.0.3-r13 version.
